File: backend/src/VeilleBoisee.Api/Auth/CollectiviteContext.cs
namespace VeilleBoisee.Api.Auth;

/// <summary>
/// Contexte de collectivité — bouchon en attendant l'authentification réelle.
/// À remplacer par une extraction depuis le JWT lors de la phase d'auth.
/// </summary>
public sealed class CollectiviteContext
{
    public string CollectiviteId { get; } = "DEV_COLLECTIVITE";

    // Liste vide = tous les signalements (mode démo — sera remplacé par les claims JWT)
    public IReadOnlyList<string> InseeCodes { get; } = Array.Empty<string>();
}

File: frontend/src/app/features/dashboard/collectivite-context.ts
import { Injectable, signal } from '@angular/core';

// Bouchon d'authentification — sera remplacé par un service JWT en phase auth
@Injectable({ providedIn: 'root' })
export class CollectiviteContext {
  readonly collectiviteId = signal('DEV_COLLECTIVITE');
  readonly label = signal('Paris (développement)');
}
